package.json

```diff
@@ -122,12 +127,14 @@
     "test:e2e:interactive": "npm run test:e2e -- --puppeteer-interactive",
     "test:e2e:ci": "npm run test:e2e -- --runInBand",
     "test:e2e:coverage": "npm run test:e2e -- --coverage --coverageDirectory=tests/coverage/e2e",
-    "test:e2e:coveralls": "npm run test:e2e -- --coverage --coverageReporters=text-lcov | coveralls",
+    "test:e2e:coveralls": "npm run test:e2e:coverage -- --runInBand --coverageReporters=json && npm run test:js:coverage-merge -- --no-js --reporter=text-lcov | coveralls",
     "test:js": "wp-scripts test-unit-js tests/js",
     "test:js:help": "wp-scripts test-unit-js --help",
     "test:js:watch": "npm run test:js -- --watch",
     "test:js:coverage": "wp-scripts test-unit-js --coverage --coverageDirectory=tests/coverage/js",
-    "test:js:coveralls": "wp-scripts test-unit-js --coverage --coverageReporters=text-lcov | coveralls",
+    "test:js:coveralls": "npm run test:js:coverage && coveralls < tests/coverage/js/lcov.info",
```

Comment:
There seems to be an issue with piping the coverage output to the `coveralls` command, failed test cases seem to pass the build. Eg (https://travis-ci.com/github/xwp/material-theme-builder-wp/jobs/317163271)